catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliCommonUtils.java

```diff
@@ -87,19 +87,22 @@ public static void setUCOEdges(Node root, Graph<Node, Edge> graph) {
         if (!visitorStack.contains(currNode)) {
           visitorStack.push(currNode);
           for (Edge edge : graph.edgesOf(currNode)) {
-
-            Node source = graph.getEdgeSource(edge);
-            Node target = graph.getEdgeTarget(edge);
-
-            // Remove if statement?
-            if (edge != null && source != null && target != null) {
-              edge.start_node = source.id;
-              edge.end_node = target.id;
-              stack.push(target);
-            }
+            processEdges(graph, stack, edge);
           }
         }
       }
     }
   }
+
+  private static void processEdges(Graph<Node, Edge> graph, Stack<Node> stack, Edge edge) {
+    Node source = graph.getEdgeSource(edge);
+    Node target = graph.getEdgeTarget(edge);
+
+    // Remove if statement?
```

Comment:
✏️  Nitpick: Can we get rid of this comment or the if statement? 
I can already feel it's starting to get stale/confusing.

Reply:
I will remove the comment